app/views/plans/_activity_fields.html.erb

```diff
@@ -1,12 +1,14 @@
 <div class='nested-fields'>
   <%
-    options = options_for_select(
-      [:monday, :tuesday, :wednesday, :thursday, :friday, :saturday, :sunday].map{ |d| [d.to_s.capitalize, d] },
+    day_options = options_for_select(
+      [:monday, :tuesday, :wednesday, :thursday, :friday, :saturday, :sunday].map { |d| [d.to_s.capitalize, d] },
       selected: f.object.day
     )
   %>
 
-  <%= f.select :day, options, label: I18n.t('activity.labels.day') %>
+  <%= f.select :day, day_options, label: I18n.t('activity.labels.day') %>
+  <%= f.time_select :start_time, label: I18n.t('activity.labels.start_time'), ignore_date: true %>
+  <%= f.time_select :end_time, label: I18n.t('activity.labels.end_time'), ignore_date: true %>
```

Comment:
```suggestion
  <div class='row'>
    <div class='col-2'>
      <%= f.time_select :start_time, label: I18n.t('activity.labels.start_time'), ignore_date: true, default: { hour: '09', minute: '00' } %>
    </div>
    <div class='col-2'>
      <%= f.time_select :end_time, label: I18n.t('activity.labels.end_time'), ignore_date: true, default: { hour: '17', minute: '00' } %>
    </div>
  </div>
```

Reply:
Can we set a default for the start and time rather than it defaulting to the current time? 9 to 5 or 9 to 12 perhaps.
Also a little styling change.